### PR TITLE
add swate tutorial videos

### PR DESCRIPTION
**Pull request by Brilator**
This is not "public", yet - out-commented in sidebar. Just to showcase embedding YouTube videos. 
Can be reached here: https://nfdi4plants.org/content/docs/swate_videos.html

### CODE REVIEW

File: src/content/docs/swate_videos.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+layout: docs
```
Comment (Freymaurer) [inline src/content/docs/swate_videos.md:2]: Most of these are outdated or the feature is currently not actively developed (Adding a checklist to your table). I don't think the current iteration of videos should be promoted.

**Review by Freymaurer — CHANGES_REQUESTED** [review]
We should propably get someone to redo the video with Swate v0.6.0 as soon as possible.

**Comment by Brilator** [comment]
I know, that's why I didn't publish. Was more for us internally. But no need to merge.

**Comment by Freymaurer** [comment]
@CMR248 ?

**Comment by Martin-Kuhl** [comment]
Dominik and I talked yesterday about embedding youtube videos to the tutorial/training section in the sitebar, currently hidden. We also noticed that the videos for SWATE (and other tools) need to be (re)done. 
I think Dominik just wanted to showcase and try it out for future material :+1:

**Comment by Freymaurer** [comment]
Then let's merge it 👍

**Review by Freymaurer — APPROVED** [review]

**Pull request merged**